README.md

```diff
@@ -60,6 +60,14 @@ PRODUCT_PACKAGES += Chrome
 
 This uses the module name. You can find the module name for a package by checking `vendor/google/build/modules/` and look at the `LOCAL_MODULE` value.
 
+### Excluding packages
+You can exclude certain packages from the list of packages associated with your selected OpenGapps variant. E.g. if you have `GAPPS_VARIANT:=stock`
```

Comment:
good catch.  I meant to use spaces there.

change made (and I also changed `GAPPS_VARIANT:=micro` to `GAPPS_VARIANT := micro`)